src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -1098,7 +1098,6 @@ internal override void CreateAsync()
                 {
```

Comment:
@daxian-dbw, your last commit had 3 failures in `PowerShell-CI-windows`
Basic Job Tests.Background pwsh process should terminate after job is done.Can clean up background pwsh process after job is done
```powershell
Expected an exception, with FullyQualifiedErrorId 'NoProcessFoundForGivenId,Microsoft.PowerShell.Commands.GetProcessCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Job\Jobs.Tests.ps1: line 349
349:             { Get-Process -Id $processId -ErrorAction Stop } | Should -Throw `
```
Basic Job Tests.Background pwsh process should terminate after job is done.Can clean up background pwsh process when job is stopped
```powershell
Expected an exception, with FullyQualifiedErrorId 'NoProcessFoundForGivenId,Microsoft.PowerShell.Commands.GetProcessCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Job\Jobs.Tests.ps1: line 365
365:             { Get-Process -Id $processId -ErrorAction Stop } | Should -Throw `
```
Basic Job Tests.Background pwsh process should terminate after job is done.Can clean up background pwsh process when job is removed
```powershell
Expected an exception, with FullyQualifiedErrorId 'NoProcessFoundForGivenId,Microsoft.PowerShell.Commands.GetProcessCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Job\Jobs.Tests.ps1: line 381
381:             { Get-Process -Id $processId -ErrorAction Stop } | Should -Throw `
```